node/trieIterators/directStakedListProcessor.go

```diff
@@ -0,0 +1,116 @@
+package trieIterators
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/api"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/vm"
+)
+
+type directStakedListProc struct {
+	*commonStakingProcessor
+	accounts           *AccountsWrapper
+	blockChain         data.ChainHandler
+	publicKeyConverter core.PubkeyConverter
+}
+
+var metachainIdentifier, _ = hex.DecodeString("ff")
```

Comment:
why not declaring a const and set it with the bytes value?

Reply:
done